src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -205,7 +209,7 @@ export default class PeopleAndRoles extends Mixins(CommonMixin) {
       addressCountry: '',
       deliveryInstructions: ''
     },
-    action: null
+    action: []
```

Comment:
Unless you've changed this, it's not an array.

If it is an array then I'd like to see it pluralized (actions).

Reply:
Definitely an Array now
Will update!

Reply:
Updated